Lucidtech.nuspec

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0"?>
 <package>
     <metadata>
-        <id>Lucidtech.LAS.Receipt</id>
+        <id>Lucidtech.Las.Receipt</id>
```

Comment:
Change package name from Receipt to something else. I suggest just using Lucidtech.Las if possible

Reply:
I agree, unless we want to use Las for other packages as well.